app/parse.py

```diff
@@ -15,8 +33,118 @@ class Product:
     num_of_reviews: int
 
 
+QUOTE_FIELDS = [field.name for field in fields(Product)]
+
+_driver: WebDriver | None = None
+
+
+def get_driver() -> WebDriver:
+    return _driver
+
+
+def set_driver(new_driver: WebDriver) -> None:
+    global _driver
+    _driver = new_driver
+
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="[%(asctime)s] - [%(levelname)8s]: %(message)s",
+    handlers=[
+        logging.StreamHandler(sys.stdout),
+        logging.FileHandler("parser.log"),
+    ]
+)
+
+
+def get_additional_info():
+    ...
+    # additional info about products
+    # not implemented
+
+
+def parse_single_product(element: WebElement) -> Product:
+    return Product(
+        title=element.find_element(
+            By.CSS_SELECTOR, ".title"
+        )
+        .get_attribute("title"),
+        description=(
+            element.find_element(
+                By.CSS_SELECTOR, ".description"
+            )
+            .text
+        ),
+        price=float(
+            element
+            .find_element(By.CSS_SELECTOR, ".price")
+            .text.replace("$", "")
+        ),
+        rating=len(element.find_elements(By.CSS_SELECTOR, ".ws-icon-star")),
+        num_of_reviews=int(
+            element
+            .find_element(By.CSS_SELECTOR, ".review-count")
+            .text.split()[0]
+        )
+    )
+
+
+def close_cookie_banner() -> None:
+    driver = get_driver()
+    close_btn = driver.find_element(By.ID, "closeCookieBanner")
+    close_btn.click()
+
+
+def load_page(url: str) -> WebDriver:
+    driver = get_driver()
+    driver.get(url)
+    time.sleep(1)
+    close_cookie_banner()
+    try:
+        load_btn = driver.find_element(
+            By.CLASS_NAME, "ecomerce-items-scroll-more"
+        )
+        logging.info(f"Loading page {url}...")
+        while load_btn.is_displayed():
+            load_btn.click()
+            time.sleep(0.1)
+    except NoSuchElementException:
+        logging.info(f"Cand load more on {url}")
```

Comment:
There is a typo in the logging message. It should be 'Cannot load more on {url}' instead of 'Cand load more on {url}'.